_policy/referees.md

```diff
@@ -1,15 +1,448 @@
+    Commission appointed by them for this purpose.
+
+### Chairperson
+
+1.  The Chairperson of the Referee Judiciary Panel must be a member of the current FIT Referee
+    Commission but does not necessarily have to be the FIT Referee Director.
+2.  The Chairperson should not be a member of the nation of the referee/s attending the judiciary
+    hearing, or involved in the incident.  
+
+### Judiciary Commission Members
+
+1.  Judiciary Panel members should be selected by the FIT Referee Director. 
+2.  Judiciary Panel members are to be respected, active referees and, if practicable, should reflect
+    a balance of the nations represented at the particular event with at least one Commission member
+    being  from the nation of the referee / Attending the judiciary hearing.  
+3.  There must be a minimum of two (2) and a maximum of four (4) Panel members plus the chairperson,
+    for the hearing to proceed.
+
+### Judiciary Hearing  
+
+1.  For a Judiciary Hearing to be convened, the FIT Referee Director must receive a written report
+    of alleged misconduct of a referee from any person associated with the game of Touch;
+2.  The FIT Referee Director, in conjunction with the FIT Referee Commission, shall decide whether
+    to, and how to, proceed based on reports received;
+3.  The FIT Board of Directors must be informed of any decision to conduct a Judiciary meeting; and
+4.  A FIT Board member should be requested to attend all FIT Referee Judiciary Hearings in an
+    observer capacity.
+
+### Powers of the Judiciary
+
+1.  If the referee is found guilty of any breach of misconduct which would be considered to
+    contravene the Referees Code of Ethics, such referee may be suspended or disqualified from
+    officiating at FIT tournaments.
+2.  The degree of any penalty and duration of any suspension or disqualification will be decided by
+    the respective Judiciary Panel.
+3.  If a FIT Referee is found guilty of misconduct (or bringing the game into disrepute), he/she
+    will be immediately stood down from the event.
+4.  The respective Member NTA must receive a report on the findings of the Judiciary Hearing.
+5.  The FIT Board of Directors must be notified in writing of such decisions and outcomes.
+
+### Appeals / Grievances
+
+1.  In accordance with natural justice and transparency, referees may appeal against decisions made
+    by the FIT Referee Judiciary Commission. Such appeals or grievances are to be dealt with in a
+    timely manner by the FIT Board of Directors or a designated Commission appointed by it.
+
+## FIT Referees Gold Badge
+
+### Outline
+
+1.  The FIT Referees Badge is designed for the most elite of referees and sits above the universally
+    acclaimed black badge and will be awarded to referees that achieve the highest level in
+    refereeing achievement. 
+2.  This badge will be introduced at the 2011 World Cup in Edinburgh, Scotland.
```

Comment:
This was 5 years ago, I think it should be removed - when it was introduced is not particularly relevant to the application.